docker/pytorch-aarch64/patches/xla-cpu-enhancements.patch

```diff
@@ -0,0 +1,1547 @@
+From db1a585d5e645c1a8a28f23b15f12119ec87767d Mon Sep 17 00:00:00 2001
```

Comment:
This file will need the appropriate (C) notice, there's no need to include the patch summary (L2-L32) really.

Reply:
It may just be simpler to bring these patches down at runtime rather than worry about adding the (C) on inclusion in this repo. This would also alleviate any concerns about the embedded (c) notices in this patchset.